idl/github.com/uber/cadence/shared.thrift

```diff
@@ -1312,6 +1313,21 @@ struct ListWorkflowExecutionsResponse {
   20: optional binary nextPageToken
 }
 
+struct ListArchivedWorkflowExecutionsRequest {
+  10: optional string domain
+  20: optional i32 maximumPageSize
+  30: optional binary nextPageToken
+  40: optional StartTimeFilter CloseTimeFilter
```

Comment:
naming?

Reply:
The actual filtering is done on closeTime. But the struct type has a bad naming. I can follow the existing patten (name the variable as `StartTimeFilter`, but internally filtering on close time), but it seems kind of weird to me. Or adding a new struct for closeTime?

Reply:
For clearness, how about creating TimeFilter (instead of using previous wrong StartTimeFilter) since this is new API doesn't have compatibility issue)
Also property name should be lower case `closeTimeFilter`